source/common/http/conn_manager_impl.cc

```diff
@@ -393,6 +393,15 @@ void ConnectionManagerImpl::resetAllStreams(
     stream.response_encoder_->getStream().removeCallbacks(stream);
     stream.onResetStream(StreamResetReason::ConnectionTermination, absl::string_view());
     if (response_flag.has_value()) {
+      // TODO(matteof): This code duplicates some of the logic in
```

Comment:
Are you planning on picking this up later?  If not I'd skip the TODO.

Reply:
ack, will be in the next push